### ISSUE TITLE
OMG

### ISSUE THREAD

**Issue by Amsu123** [benqSzaw/BlackWorld_JD#6]
THIS GAME IS AMAZING I LOVE THIS GAME. I CAN'T STOP PLAYING THIS GAME!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by nejfyn** [comment]
I AGREE!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by nejfyn** [comment]
<3

**Comment by nejfyn** [comment]
^
     |
     |
     |
     |
     |
  GAY
